cowprotocol/accounting/fees/protocol_fees_by_order_class_3703312.sql

```diff
@@ -0,0 +1,44 @@
+with
+protocol_fees_collected as (
+    select --noqa: ST06
+        order_uid, --noqa: RF03
```

Comment:
it honestly sounds weird to me that we need a `noqa` comment on every line here. What exactly is the problem? Can this be resolved by slightly renaming of the select clauses?

Reply:
I believe the complaint is due to the dictionary structure of the `data` part of `cowswap.raw_order_rewards`. This will go away with a follow-up PR that will make the query use the new raw order data table.